### ISSUE TITLE
Weird pre-train loss

### ISSUE THREAD

**Issue by BubvieyKevin** [thunlp/LLaVA-UHD#9]
Hello, congratulations for publishing a good paper and thank you for your excellent work.
During the pretraining and finetuning process, I noticed that the loss remained at 0 consistently. Upon checking the image tensor, I found that many tensors were also 0. 
Is this normal? What are your thoughts on this phenomenon?  How can I address this issue?

**Comment by BubvieyKevin** [comment]
We are currently addressing this issue, and the code will be ready within a few days.

**Comment by xrorrim** [comment]
Thanks for report this issue and we have fixed this problem in the latest version of code.

**Comment by guozonghao96** [comment]
Our repository has been fully improved, and almost all bugs have been eliminated. For details, please refer to the main branch and the LLaVA-UHD v1 branch. This issue is now closed. If there are any new problems, feel free to open a new issue.
